src/main/java/frc/robot/subsystems/Transfer.java

```diff
@@ -47,7 +48,27 @@ public void setFeederNeutralOutput() {
   }
 
   public boolean getGamePieceCollected() {
-    return (constTransfer.NOTE_SENSOR_INVERT) ? !noteSensor.get() : noteSensor.get();
+    // return (constTransfer.NOTE_SENSOR_INVERT) ? !noteSensor.get() :
```

Comment:
You can make this code a bit cleaner by moving the value of the note sensor into it's own line:

boolean noteSensorValue = (constTransfer.NOTE_SENSOR_INVERT) ? !noteSensor.get() : noteSensor.get();
return (noteSensorValue  || hasGamePiece);

Reply:
I think the implementation of should work though!! We should test it before we merge it in